docs/docs/quickstart.mdx

```diff
@@ -4,61 +4,38 @@ hide_title: false
 sidebar_position: 2
 ---
 
-**Ready to give Apache Superset a try?** This quickstart will help you run Superset on your local machine in
-**5 simple steps**. It assumes that you have [Docker](https://www.docker.com) installed.
+**Ready to give Apache Superset a try?** This quickstart guide will help you
+get up and running on your local machine in **3 simple steps**. Note that
+it assumes that you have [Docker](https://www.docker.com),
+[docker-compose](https://docs.docker.com/compose/), and
+[git](https://git-scm.com/) installed.
 
-### 1. Get Superset
-To get started, set the `SUPERSET_VERSION` environment variable with the latest Superset version.
-[Click here](https://github.com/apache/superset/releases) to check the latest version.
-
-```
-$ export SUPERSET_VERSION=<latest_version>
-```
-
-Pull the Superset image from Docker Hub:
+:::caution
+While we recommend using `docker-compose` for a quick start in a sandbox-type
+environment and for other development-type use cases, **we
+do not recommend this setup for production**. For this purpose please
+refer to our
+[Installing on Kubernetes](/docs/installation/running-on-kubernetes/)
+page.
+:::
 
-```
-$ docker pull apache/superset:$SUPERSET_VERSION
-```
+### 1. Get Superset
 
-### 2. Start Superset
-:::tip
-Note that some configuration is mandatory for Superset in order to start. In particular, Superset will not start without
-a user-specified value of `SECRET_KEY` in a Superset configuration file or `SUPERSET_SECRET_KEY` as an environment variable.
-Please see [Configuring Superset](/docs/installation/configuring-superset/) for more details.
-:::
-```
-$ docker run -d -p 8080:8088 \
-             -e "SUPERSET_SECRET_KEY=$(openssl rand -base64 42)" \
-             -e "TALISMAN_ENABLED=False" \
-             --name superset apache/superset:$SUPERSET_VERSION
+```bash
+$ git clone git@github.com:apache/superset.git
 ```
 
-### 3. Create an account
-```
-$ docker exec -it superset superset fab create-admin \
-              --username admin \
-              --firstname Admin \
-              --lastname Admin \
-              --email admin@localhost \
-              --password admin
-```
+### 2. Start the latest official release of Superset
 
-### 4. Configure Superset
 ```
-$ docker exec -it superset superset db upgrade &&
-         docker exec -it superset superset load_examples &&
-         docker exec -it superset superset init
+$ docker-compose -f docker-compose-image-tag.yml up
```

Comment:
This command won't work unless the user is already in Superset directory, should be `docker-compose -f superset/docker-compose-image-tag.yml up` or `cd superset && docker-compose -f docker-compose-image-tag.yml up`